src/gs-details-page.c

```diff
@@ -2265,6 +2265,16 @@ gs_details_page_network_available_notify_cb (GsPluginLoader *plugin_loader,
 	gs_details_page_refresh_reviews (self);
 }
 
+static void
```

Comment:
These callbacks need to return `FALSE` (or `GDK_EVENT_PROPAGATE`/however it's called)